packages/backend/src/models/SpaceModel.ts

```diff
@@ -419,12 +420,17 @@ export class SpaceModel {
                 'saved_queries_versions',
                 `saved_queries.saved_query_id`,
                 `saved_queries_versions.saved_query_id`,
-            )
+            ) // TODO remove
```

Comment:
Most models that use `saved_queries_versions` need to be updated as well:
- ResourceViewItemModel
- SpaceModel
- SavedChartModel

Let's create a new ticket for these changes and link to this ticket/PR.